docker/mock-pay/Dockerfile

```diff
@@ -0,0 +1,10 @@
+
+FROM badouralix/curl-jq:latest as build
+
+RUN curl https://raw.githubusercontent.com/alphagov/pay-publicapi/master/openapi/publicapi_spec.json --output openapi-spec.json
+RUN cat openapi-spec.json | jq '.paths."/v1/payments/{paymentId}".get.responses."200".content."application/json".examples = { "full-fee": { value: { amount: 8200, payment_id: "hu20sqlact5260q2nanm0q8u93", reference: "your-reference" } }, "half-fee": { value: { amount: 4100, payment_id: "hu20sqlact5260q2nanm0q8u93", reference: "your-reference" } }}' > mktemp && mv mktemp openapi-spec.json
```

Comment:
ha, this is a fun way of adding it

why `> mktemp && mv mktemp openapi-spec.json` and not `> openapi-spec.json` at the end? Or `> modified-openapi-spec.json` and then use that in the next step

Reply:
That works too - some leftover copy-paste from SO.